README.md

```diff
@@ -197,18 +197,16 @@ You can see the build statuses at
 
 #### Running all tests
 
-To execute all the tests, run this command at the root of the Solidus project
-to run specs for all projects:
+To execute all the tests for each project, just run `rake` at the top level of
+the project.
 
-```shell
-bash build.sh
-```
-
-This runs using postgresql by default, but can be overridden by specifying
-`DB=sqlite` or `DB=mysql` in the environment.
+This runs using sqlite by default, but can be overridden by specifying
+`DB=postgresql` or `DB=mysql` in the environment.
 
-[PhantomJS](http://phantomjs.org/) is required for the frontend and backend
-test suites.
+A recent version of chrome and chromedriver is required to run the frontend and
+backend test suites. If you don't have chromedriver,
+[`gem install chromedriver-helper`](https://github.com/flavorjones/chromedriver-helper)
```

Comment:
Using the gem actually conflicts with having it globally installed. I would to see us promoting installing it system wide via package managers instead of via Ruby gems